.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,38 @@
+4. See error
+
+### Expected Behavior
+A clear description of what you expected to happen.
+
+### Actual Behavior
+What actually happened instead.
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
+### System Information:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)